content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,67 @@
+  explicitly select which CPUs to reserve instead of letting the kubelet pick them up automatically.
+- [report the exclusively allocated CPUs](https://github.com/kubernetes/kubernetes/pull/97415) to containers, much like is already done for devices,
+  using the kubelet-local [podresources API](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources).
+- [optimize the usage of system resources](https://github.com/kubernetes/kubernetes/pull/101771), eliminating unnecessary sysfs changes.
+
+The CPU Manager reached the point on which it "just works", so in Kubernetes v1.26 it is graduating to GA.
+
+### Stable does not mean inflexible: customization options for CPU Manager
+
+The CPU Manager supports two operation modes, configured using its _policies_. With the `none` policy, the CPU Manager allocates CPUs to containers
+without any specific constraint except the quota set in the Pod Spec, if present. With the `static` policy, when the pod is in the Guaranteed QoS Class and
+finally when the amount of CPU requested by containers is integral the CPU Manager allocates CPUs exclusively.
+
+This simple operational model served the user base pretty well, but as the CPU Manager matured more and more, users started to look at more elaborate use
+cases and how to better support them.
+
+Rather than add more policies, the community realized that pretty much all the novel use cases are some variation of the behavior enabled by the `static`
+CPU Manager policy. Hence, it was decided to add [options to tune the behavior of the static policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2625-cpumanager-policies-thread-placement#proposed-change).
+The options have a varying degree of maturity, like any other Kubernetes feature, and in order to be accepted, each new option provides a backward
+compatible behavior when disabled, and to document how to interact with each other, should they interact at all.
+
+This enabled the kubernetes project to graduate to GA the CPU Manager core component and core CPU allocation algorithms to GA,
+while also enabling a new age of experimentation in this area.
+In kubernetes v1.26, the CPU Manager [supports three different policy options](docs/tasks/administer-cluster/cpu-management-policies.md#static-policy-options):
+
+- `full-pcpus-only` restrict the CPU Manager core allocation algorithm to full physical cores only, reducing the noisy neighbors issues.
+- `distribute-cpus-across-numa` drive the CPU Manager to evenly distribute CPUs across NUMA nodes in cases where more than one NUMA node is required to satisfy the allocation.
+- `align-by-socket` make the CPU Manager consider CPUs aligned at the socket boundary, instead of NUMA Node boundary, when deciding how to allocate CPUs to a container.
```

Comment:
How about:
```suggestion
`full-pcpus-only`
: restrict the CPU Manager core allocation algorithm to full physical cores only, reducing noisy neighbor issues from hardware technologies that allow sharing cores.

`distribute-cpus-across-numa`
: drive the CPU Manager to evenly distribute CPUs across NUMA nodes, for cases where more than one NUMA node is required to satisfy the allocation.

`align-by-socket`
: change how the CPU Manager allocates CPUs to a container:  consider CPUs to be aligned at the socket boundary, instead of NUMA node boundary.
```

Reply:
seems nicer, using your suggestion